hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -161,7 +211,7 @@ type Test =
 --
 newtype TestT m a =
   TestT {
-      unTest :: ExceptT Failure (Lazy.WriterT [Log] m) a
+      unTest :: ExceptT Failure (Lazy.WriterT (PropResult, [Log]) m) a
```

Comment:
@jystic, perhaps we can take this as-is for now and later refactor this tuple to some data type, e.g. `Journal`. My only concern is that this can easily end up being a triple, quadruple, etc. in the future.